main.py

```diff
@@ -90,7 +98,7 @@ def execute_consistency(self):
                         with circuit.lock:
                             circuit.deploy()
         for circuit_id in stored_circuits:
-            log.info(f"EVC found in storehouse but unloaded {circuit_id}")
+            log.info(f"EVC found in mongodb but unloaded {circuit_id}")
```

Comment:
Future opportunity (low prio considering the strategy we went for):

This is looping over a collection calling `_load_evc`, and every evc will be upserted again after it gets deactivated, in the future this can be done in a single DB bulk_write instead of N operations.

Reply:
Yes, totally agree with you, @viniarck! I suggest we keep working on improving this using bulk updates as we feel the need. Thanks for pointing that out.